aboutlibraries-core/src/androidMain/kotlin/com/mikepenz/aboutlibraries/util/AndroidParser.kt

```diff
@@ -4,9 +4,12 @@ import android.util.Log
 import com.mikepenz.aboutlibraries.entity.*
 import org.json.JSONObject
 
-actual fun parseData(json: String): Result {
+actual fun parseData(json: String, recoverable: Boolean): Result {
```

Comment:
Based on the adjustments, let's not have a `recoverable` flag for this behaviour, and let it happen automatically. 

Especially as this is certainly an non-anticipated behaviour - and practically is not meant to happen (given the meta data is meant to contain that information)

This will also keep the API surface more simple, and does not require a new configuration for the exposed API.

Can you please adjust this

Reply:
removing :)